### ISSUE TITLE
I think that 'thunk' is not working well

### ISSUE THREAD

**Issue by dontcryme** [felixmaker/thunk#6]
If i just follow the instruction below :

```
cargo new build_for_xp
cd build_for_xp
thunk --os xp --arch x86 -- --release
```

That release binary file is only works on my current os(windows 11 x64).

If i using that binary file in window xp then i got ".... .exe is not a valid win32 application" error.

Could you have been try execute that release file in windows xp?

**Comment by felixmaker** [comment]
Hi,

I have noticed the problem. Thank you for your reporting!

For now, you can add `--subsystem console` to make thunk work.

```
thunk --os xp --arch x86 --subsystem console -- --release
```

I will make `--subsystem` default to `console` in next version.

**Comment by felixmaker** [comment]
I have fixed it in v0.2.2. It should work now.

**Comment by dontcryme** [comment]
@felixmaker 
Thank you!
It works very well.
